modules/ROOT/pages/dataweave-formats-multipart.adoc

```diff
@@ -280,6 +280,7 @@ The Multipart format accepts properties that provide instructions for reading in
 | Parameter | Type | Default | Description
 | `boundary` | `String` | `null` | The multipart `boundary` value. A string to
 delimit parts.
+|`defaultContentType`  |`String`| no default set for this property | Sets the default Content-Type to use on parts of the `multipart/*` format. When set, this property takes precedence over the https://docs.mulesoft.com/mule-runtime/4.3/dataweave-system-properties[system property setting] for `com.mulesoft.dw.multipart.defaultContentType`. _Introduced in DataWeave 2.3 (2.3.0-20210720) for the August 2021 release of Mule 4.3.0-20210719._
```

Comment:
Should the default column be blank instead?
Or maybe "(None)"
Or "No default value is set for this property."
Or "This property has no default value."

Reply:
sometimes "None" is the default, so I wanted to avoid any possible . 
```suggestion
|`defaultContentType`  |`String`| This property has no default value. | Sets the default Content-Type to use on parts of the `multipart/*` format. When set, this property takes precedence over the https://docs.mulesoft.com/mule-runtime/4.3/dataweave-system-properties[system property setting] for `com.mulesoft.dw.multipart.defaultContentType`. _Introduced in DataWeave 2.3 (2.3.0-20210720) for the August 2021 release of Mule 4.3.0-20210719._
```